Blahgger/Views/Account/Register.cshtml

```diff
@@ -0,0 +1,62 @@
+﻿@model Blahgger.Models.RegisterUser
+
+@{
+    ViewBag.Title = "Register";
+}
+
+<h2>Register</h2>
+
+
+@using (Html.BeginForm()) 
+{
+    @Html.AntiForgeryToken()
```

Comment:
Again, don't forget to add the `ValidateAntiForgeryToken` attribute to the POST action method.